lib/ansible/parsing/dataloader.py

```diff
@@ -290,12 +290,12 @@ def path_dwim_relative_stack(self, paths, dirname, source, is_role=False):
             for path in paths:
                 upath = unfrackpath(path, follow=False)
                 b_upath = to_bytes(upath, errors='surrogate_or_strict')
-                b_mydir = os.path.dirname(b_upath)
+                b_cwd = os.path.dirname(b_upath)
```

Comment:
this shoudl never be 'cwd', but basedir  .. which should be 'playbook dir' or dervied